### DISCUSSION TITLE
Contributing to improving docs

### DISCUSSION THREAD

**Discussion by railBitsAndHex** [supabase/supabase#7329] [Questions] [answered]
Hi, would like to contribute to improving the [Supabase Reference Documentation](https://supabase.com/docs/reference) .

May I know how can I contribute? (E.g. Any helpful resources containing details on how to contribute to this particular section of the docs etc.)

**Comment by GaryAustin1** [comment, marked as answer]
This is the process for doing a PR on the docs:
https://github.com/supabase/supabase/tree/master/web

**Reply by railBitsAndHex** [reply]
Got it. Thanks a lot!
